widget/calendar.go

```diff
@@ -27,14 +29,18 @@ var minCellContent = NewLabel("22")
 // Since: 2.6
 type Calendar struct {
 	BaseWidget
-	currentTime time.Time
+	displayedMonth time.Time
 
 	monthPrevious *Button
 	monthNext     *Button
 	monthLabel    *Label
 
 	dates *fyne.Container
 
+	SelectedDate time.Time // currently selected date
+
+	// TODO discuss a config variable WeekStart time.Weekday to override week's first day
```

Comment:
See widget/locale.go

Reply:
Yes, I saw the mecanism used to selected first week day.
But I thought it would good for people like me, in obscure non-typical countries, to be able to modify first week day

Reply:
It could be an additional Fyne setting I guess, if you want to override what we get from the system..

Reply:
OK, so I'll make a feature request to track this